components/camel-aws/camel-aws2-cw/pom.xml

```diff
@@ -80,9 +80,5 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.camel</groupId>
-            <artifactId>camel-health</artifactId>
-        </dependency>
```

Comment:
This is needed for health check to work

Reply:
It's a dup. It's already defined above in the pom.

Reply:
Well spotted. I shouldn't do review from the bed